recipe/conda_build_config.yaml

```diff
@@ -137,6 +137,8 @@ libwebp:
   - 0.5
 libxml2:
   - 2.9
+lz4-c:
```

Comment:
Forgot `-` is not allowed here. So one must use `_`, which `conda-build` treats the same as `-`. Fixing in PR ( https://github.com/conda-forge/conda-forge-pinning-feedstock/pull/24 ).